learn/pkg/learn/renderer.go

```diff
@@ -180,6 +181,50 @@ func (s *evySource) RenderHTML(buf *bytes.Buffer) {
 	buf.WriteString("</code></pre>\n")
 }
 
+func newTxtarContent(filename string, resultType ResultType) (Renderer, error) {
+	b, err := os.ReadFile(filename)
+	if err != nil {
+		return nil, err
+	}
+	archive := txtar.Parse(b)
+	m := map[string]bool{}
+	for _, file := range archive.Files {
+		content := string(file.Data)
+		if m[content] {
+			return nil, fmt.Errorf("%w: found duplicate content", ErrBadTxtar)
+		}
+		m[content] = true
+	}
+	return &txtarContent{
+		data:       b,
+		ResultType: resultType,
+		archive:    archive,
+	}, nil
+}
+
+type txtarContent struct {
+	data       []byte
+	ResultType ResultType
+	archive    *txtar.Archive
+}
+
+// RenderOutput executes.
+func (*txtarContent) RenderOutput() string {
+	return "*** txtar Content ERROR ***"
+}
+
+// RenderHTML prints evy source code.
+func (*txtarContent) RenderHTML(buf *bytes.Buffer) {
+	buf.WriteString("*** txtar Content ERROR ***")
+}
+
+func (t *txtarContent) Archive() *txtar.Archive {
+	if t.archive == nil {
+		t.archive = txtar.Parse(t.data)
+	}
```

Comment:
How come this is here? `newTxtarContent` sets the field so it should not be nil.

Reply:
that was probably a left over from the initial implementation, apologies. I've removed it.